src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsWriter.cs

```diff
@@ -34,7 +34,7 @@ public TdsWriter(TdsStream stream)
         /// Writes short value to out buffer
         /// </summary>
         /// <param name="v">Value to write</param>
-        /// <param name="isAsync">Is Async caller method</param>
+        /// <param name="isAsync">Whether caller method is executing asynchronously.</param>
```

Comment:
I like how you have created these comments uniformly. 
Is there some tool that you are using? Or did you manually copy paste these blurbs to make them look alike?

Reply:
I wish there was a tool :) Manually here.